goldstone/core/models.py

```diff
@@ -2634,3 +2636,70 @@ def integration(cls):
     def resourcetype(cls):
 
         return "routers"
+
+
+class DefinedSearch(models.Model):
```

Comment:
Your last commit mentioned that adding DefinedSearch to core/admin.py broke some tests.  That's very puzzling.  What was the error?